front/src/applications/stdcm/views/OSRDStdcmResults.tsx

```diff
@@ -93,6 +135,23 @@ const OSRDStcdmResults = () => {
             )}
           </div>
         </div>
+        <div>
+          {!simulationSheetloading && stdcmResults && (
```

Comment:
I think the simulationSheetLoading is not necessary, and stdcmResults is enough
```suggestion
          {stdcmResults && (
```

Reply:
done